inc/signup_notification/namespace.php

```diff
@@ -160,3 +185,340 @@ function altis_signup_blog_notification( $domain, $path, $title, $user_login, $u
+ * replace it with your own notification behavior.
+ *
+ * Filter {@see 'wpmu_signup_user_notification_email'} and
+ * {@see 'wpmu_signup_user_notification_subject'} to change the content
+ * and subject line of the email sent to newly registered users.
+ *
+ * @since MU (3.0.0)
+ *
+ * @param string $user_login The user's login name.
+ * @param string $user_email The user's email address.
+ * @param string $key        The activation key created in wpmu_signup_user()
+ * @param array  $meta       Optional. Signup meta data. Default empty array.
+ * @return bool
+ */
+function altis_signup_user_notification( $user_login, $user_email, $key, $meta = [] ) {
+	/**
+	 * Filters whether to bypass the email notification for new user sign-up.
+	 *
+	 * @since MU (3.0.0)
+	 *
+	 * @param string $user_login User login name.
+	 * @param string $user_email User email address.
+	 * @param string $key        Activation key created in wpmu_signup_user().
+	 * @param array  $meta       Signup meta data. Default empty array.
+	 */
+	if ( ! apply_filters( 'wpmu_signup_user_notification', $user_login, $user_email, $key, $meta ) ) {
+		return false;
+	}
+
+	// Patch error in wp-user-signups passing through signup key.
+	if ( class_exists( 'WP_Signup_Query' ) && empty( $key ) ) {
+		$query = new WP_Signup_Query( [
+			'user_email' => $user_email,
+			'user_login' => $user_login,
+		] );
+		$key = $query->signups[0]->activation_key ?? '';
+	}
+
+	$user            = get_user_by( 'login', $user_login );
+	$switched_locale = $user && switch_to_user_locale( $user->ID );
+
+	// Send email with activation link.
+	$admin_email = get_site_option( 'admin_email' );
+
+	if ( '' === $admin_email ) {
+		$admin_email = 'support@' . wp_parse_url( network_home_url(), PHP_URL_HOST );
+	}
+
+	$admin_email = apply_filters( 'wp_mail_from', $admin_email );
+
+	$from_name       = ( '' !== get_site_option( 'site_name' ) ) ? esc_html( get_site_option( 'site_name' ) ) : 'Altis';
+	$message_headers = "From: \"{$from_name}\" <{$admin_email}>\n" . 'Content-Type: text/plain; charset="' . get_option( 'blog_charset' ) . "\"\n";
+	$message         = sprintf(
+		/**
+		 * Filters the content of the notification email for new user sign-up.
+		 *
+		 * Content should be formatted for transmission via wp_mail().
+		 *
+		 * @since MU (3.0.0)
+		 *
+		 * @param string $content    Content of the notification email.
+		 * @param string $user_login User login name.
+		 * @param string $user_email User email address.
+		 * @param string $key        Activation key created in wpmu_signup_user().
+		 * @param array  $meta       Signup meta data. Default empty array.
+		 */
+		apply_filters(
+			'wpmu_signup_user_notification_email',
+			/* translators: New user notification email. %s: Activation URL. */
+			__( "To activate your user, please click the following link:\n\n%s\n\nAfter you activate, you will receive *another email* with your login." ),
```

Comment:
Could this be "To activate your user account..."? To be a little more friendly?

Reply:
Pure lift and shift from WP core. I don’t disagree though. We can make this update

Reply:
It would be nice (and consistent with the mention of "administrator account" elsewhere). But we would lose the core translation, as we are relying on them throughout this file.

Reply:
Ah of course, good point!